doc/api-changes/correct_root_2016-08-15.rst

```diff
@@ -0,0 +1,6 @@
+`correct_root` supports uids
```

Comment:
You have discovered an old directory full of API changes snippets that was never actually included in the rendered HTML documentation. Someone else was also (understandably) confused by this and I recently deleted `doc/api-changes` in a pull request that came after you created this branch. Use `doc/source/whats_new/v0.9.0.txt`.